src/server/lib/templateDropdowns.json

```diff
@@ -218,14 +219,14 @@
     "Other (please specify)"
   ],
   "Subrecipient": [
-    "Subrecipient",
-    "Contractor",
-    "Beneficiary"
+    "-Subrecipient;",
```

Comment:
@jcomeau1 what's up with these `-` and `;` at the front/end of these?

Reply:
Yeah, we can definitely do normalization during output!

Any insight into why there are hyphens _and_ semicolons?